book/website/collaboration/new-community/new-community-guide.md

```diff
@@ -0,0 +1,90 @@
+# Guide to Planning a Community
+
+What if you started a project with a thought like, “I have this great idea that I want to try on this public data!”?
+There is nothing to worry about if you’re the only one working on it.
+However, if you want to develop this project - you become responsible for making people feel included in your project.
+
+As a 'project lead', you want to set up a welcoming and inclusive environment and create the first set of visions and goals for your collaborators.
+You can not assume that everyone you collaborate with knows what is expected of them when they start to work with others on your project.
+Therefore it’s important to set the right expectations from the beginning for your community, even though you might not have planned on having one (see more details: {cite}`Sharan2020Apr`).
+
+## A checklist for planning collaboration in your project
+
+The checklist below will help you in making the process of establishing collaboration in your research project thoughtfully in a structured manner.
+
+The practices listed here are derived from and limited by the experiences of the authors who participate in several successful Open Research communities and lead community-driven projects such as [The Carpentries](https://carpentries.org), [Mozilla Open Leaders](https://mozilla.github.io/open-leadership-training-series/), [Open Life Science](openlifesci.org/) and _The Turing Way_.
+While reading this chapter, please be aware that you may need to make adjustments for the projects that may be very different in nature (for example, not entirely open source).
+
+### 1. Choose a communication platform
+
+- When leading an open project, use collaborative and open platforms such as [GitHub](http://github.com/).
```

Comment:
```suggestion
- When leading an open project, use collaborative and open platforms such as [GitHub](http://github.com/) or [GitLab](https://about.gitlab.com/).
```

Just to add an alternative 🙃

Reply:
(I added this because there was an alternative to Slack included, but the rest of the subchapter focuses on GitHub, so you don't also need to include GitLab.)